Source/WebCore/html/ValidatedFormListedElement.cpp

```diff
@@ -0,0 +1,494 @@
+/*
+ * Copyright (C) 2022 Apple Inc. All rights reserved.
```

Comment:
We should probably need to retain the old copyright given we're not writing this from scratch.

Reply:
Retained in [67e744a](https://github.com/WebKit/WebKit/pull/4988/commits/67e744aff523eae6a7b3c73ac808978d5548d7e4).